.gitattributes

```diff
@@ -0,0 +1 @@
+*.npy filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why do we use this?

Reply:
Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or GitHub Enterprise. 
In each Git repository where we want to use Git LFS, we have to select the file types we'd like Git LFS to manage. In this case, I wanted to use .npy files. I also had to add .gitattributes to track my files.
Documentation link: https://git-lfs.com/